arviz/stats/stats.py

```diff
@@ -1353,42 +1363,26 @@ def summary(
     # Combine metrics
     metrics: List[xr.Dataset] = []
     metric_names: List[str] = []
-    if extend:
-        metrics_: Tuple[xr.Dataset, ...]
+    if extend and kind in ["all", "diagnostics"]:
+        metrics_: Tuple[xr.Dataset, ...] = (
+            mcse_mean,
+            mcse_sd,
+            ess_bulk,
+            ess_tail,
+            r_hat,
+        )
         metrics_names_: Tuple[str, ...] = (
-            "mean",
-            "sd",
-            f"hdi_{100 * alpha / 2:g}%",
-            f"hdi_{100 * (1 - alpha / 2):g}%",
             "mcse_mean",
             "mcse_sd",
             "ess_bulk",
             "ess_tail",
             "r_hat",
         )
-        if kind == "all":
-            metrics_ = (
-                mean,
-                sd,
-                hdi_lower,
-                hdi_higher,
-                mcse_mean,
-                mcse_sd,
-                ess_bulk,
-                ess_tail,
-                r_hat,
-            )
-        elif kind == "stats":
-            metrics_ = (mean, sd, hdi_lower, hdi_higher)
-            metrics_names_ = metrics_names_[:4]
-        elif kind == "diagnostics":
-            metrics_ = (mcse_mean, mcse_sd, ess_bulk, ess_tail, r_hat)
-            metrics_names_ = metrics_names_[4:]
```

Comment:
As we are taking care of the situation when `kind=="stats"` already and when kind equals `"diagnostics"` or `"all"` then we will have to add these quantities in both the cases, therefore I removed the if/elif conditions here. Can you please review this part.

Reply:
this should be removed yes, but there are still some changes needed to recover the "original" output